From 8fab07c7835fd24e4fe1fba3eeac9a1ed005b9e9 Mon Sep 17 00:00:00 2001
From: Michael Pauzner <mpauzner@habana.ai>
Date: Thu, 24 Apr 2025 15:45:37 +0300
Subject: [PATCH] [SW-227132] gaudi3: define d1 full addr

Bare-metal ppboot doesn't use die1 ARCs.
However, the compilation of preboot with arc0 and arc2,
will fail. The similar problem has been solved in SW-205285.
Add these macros, to fix the compilation.

Note: revise preboot build.

Change-Id: I2cc74ebf1b8ffd6051e26a3005b13e983fcdad8b
Signed-off-by: Michael Pauzner <mpauzner@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index 4d2b9b8..aa68386 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -50,7 +50,9 @@
 #define ARC2_DCCM_FULL_BASE		0x300007FD30E0000ull
 #define ARC2_DCCM_SIZE			SZ_32K
 
+#define D1_ARC0_ICCM_FULL_BASE		0x300007FDB000000ull
 #define D1_ARC1_ICCM_FULL_BASE		0x300007FDB040000ull
+#define D1_ARC2_ICCM_FULL_BASE		0x300007FDB0C0000ull
 
 #define ARC1_OFFLOAD_DCCM_FULL_BASE    0x0300007FDB0E0000ull
 #define ARC1_OFFLOAD_DCCM_SIZE         SZ_32K
